### DISCUSSION TITLE
Usgeocoder Recurring Issue

### DISCUSSION THREAD

**Discussion by cyber0567** [community/community#124080] [Other Feature Feedback, Questions, & Ideas]
### Usaepay Recurring Issue in PHP

I am in development of the PHP(Cake-PHP) and I am trying to integrate the usepay for payment.
For first payment, I am getting the token using "cc:save".
For recurring, I don't have CVV for users and I want to do recurring payment without CVV check.
What is the solution?

**Comment by brandonsoft** [comment]
You can use different key for initial payment and recurring.

And you can set that first key required CVV and next one does not require CVV.
So for initial payment, please use first key and for recurring payment, please use second key.

I have figured out this problem using above way.

**Comment by cyber0567** [comment]
Thanks.
This works well.

**Comment by ebndev** [comment]
Hi @maksimsoftdev & @awesomedev77, thanks for being a part of the GitHub Community!

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
